index.html

```diff
@@ -0,0 +1,98 @@
+                <p class="flex110 primary fontsize20 fontWeight700 lineHeight15 fs200">Meeting</p>
+                <p class="flex330 white lineHeight15 fs80">Listen to the CEO of our league opening meeting</p>
+            </div>
+            <div class="programStyle flex flexColumn">
+                <i class="width20 fa-solid fa-flag-checkered fa-2x"></i>
+                <p class="flex110 primary fontsize20 fontWeight700 lineHeight15 fs200">Speed racing</p>
+                <p class="flex330 white lineHeight15 fs80">Follow the fullspeed races on the amazing maps created by Skodriger20</p>
+            </div>
+            <div class="programStyle flex flexColumn">
+                <i class="width20 fa-solid fa-people-group fa-2x"></i>
+                <p class="flex110 primary fontsize20 fontWeight700 lineHeight15 fs200">Fan contest</p>
+                <p class="flex330 white lineHeight15 fs80">Compete against the other fans in order to win gaming gear from our sponsors</p>
+            </div>
+            <div class="programStyle flex flexColumn">
+                <i class="width20 fa-solid fa-flag-checkered fa-2x"></i>
+                <p class="flex110 primary fontsize20 fontWeight700 lineHeight15 fs200">RPG racing</p>
+                <p class="flex330 white lineHeight15 fs80">Follow the RPG races on the world`s hardest maps</p>
+            </div>
+            <div class="programStyle flex flexColumn">
+                <i class="width20 fa-solid fa-trophy fa-2x"></i>
+                <p class="flex110 primary fontsize20 fontWeight700 lineHeight15 fs200">Awards</p>
+                <p class="flex330 white lineHeight15 fs80">Awards ceremony and closing meeting</p>
+            </div>
+        </div>
+        <div class="container1 flex justifyCenter">
+            <button type="button" class="white fontSize20 joinButton displayHideBig">Join the event</button>
+            <button type="button" class="white seeProgram displayNoneSmall">SEE THE WHOLE PROGRAM</button>
+        </div>
+    </section>
+    <section class="playerArea">
+    </section>
+    <section class="backgroundGray displayNoneSmall">
+        <p class="textCenter fontSize24 lineHeight15 partnersHome tertiary">Partners</p>
+        <ul class="flex flexWrap justifyCenter listNone partnersList">
+            <li class="sponsor fontSize20 fontWeight700 tertiary ">Google</li>
+            <li class="sponsor fontSize20 fontWeight700 tertiary">Microsoft</li>
+            <li class="sponsor fontSize20 fontWeight700 tertiary">Facebook</li>
+            <li class="sponsor fontSize20 fontWeight700 tertiary">Audi</li>
+            <li class="sponsor fontSize20 fontWeight700 tertiary">Asus</li>
+        </ul>
+    </section>
+    <section class="grid3 displayNoneSmall">
+        <img src="./img/RSL.png" alt="logo">
+        <div class="flex flexAlignCenter lineHeight15">Romanian Esports League 2022<br> All rights reserved</div>
+    </section>
```

Comment:
- [ ] Kindly make sure you use a `footer` tag instead of a `section` tag 🙏 . This would make your webpage more accessible